copilot-chat-prompt-mode.el

```diff
@@ -54,52 +54,59 @@
   :lighter " Copilot Chat Prompt"
   :keymap copilot-chat-prompt-mode-map)
 
-(defun copilot-chat--write-buffer(data save &optional buffer)
+(defun copilot-chat--write-buffer(instance data save &optional buffer)
```

Comment:
It is easier to deal with this now than later, as it is just a matter of putting appropriate line breaks in the document.